### ISSUE TITLE
En tant que membre, je peux aller vers le point d'origine de la notification.

### ISSUE THREAD

**Issue by rbourrier** [assomaker/dailylife#50]
J'arrive pas à exprimer clairement mon idée. L'objectif c'est par exemple si l'on clique sur une notification automatique d'une fiche événement, cliquer dessus nous renvoie vers la Fiche Evenement

**Comment by remipichon** [comment]
Chaque notification peut avoir un champ URL source qui sera saisi automatiquement pour les notifications automatiques, comme celles des comptes perso qui redirigent vers son compte perso, celles des deadlines qui redirigent vers la liste des fiches anims non validées.. 

Est ce cela dont tu parles ?

**Comment by rbourrier** [comment]
Yep, mais du coup comme on parle pas de la structure de données dans les
issues je ne sais pas comment en parler

**Comment by remipichon** [comment]
Je vois, ce nom ira. Sauf si quelqu'un trouves une autre formulation. Nous
y reviendrons un peu plus tard

**Comment by cpalliere** [comment]
Moi j'avais compris l'intitulé donc ça me va

**Comment by remipichon** [comment]
C'est bon alors
